.github/workflows/.cargo/audit.toml

```
@@ -0,0 +1,5 @@
+# All of the options which can be passed via CLI arguments can also be permanently specified in this file.
+# A good example is at https://docs.rs/crate/cargo-audit/0.10.0/source/audit.toml.example
+
+[advisories]
+ignore = ["RUSTSEC-2020-0071", "RUSTSEC-2020-0159"] # advisory IDs to ignore
```

We should not ignore the vulnerability with time since that has a fix.

```suggestion
ignore = ["RUSTSEC-2020-0159"] # advisory IDs to ignore
```

---

Including cargo raises both alerts. What is the fix for RUSTSEC-2020-0071